### PR TITLE
Added files to wiki

### PR DESCRIPTION
**Pull request by VLiamin**

### CODE REVIEW

File: src/FileService.Business/Commands/File/CreateFilesCommand.cs [hunk 1]
```diff
@@ -41,17 +44,30 @@ public class CreateFilesCommand : ICreateFilesCommand
...
+      {
+        (ProjectStatusType projectStatus, ProjectUserRoleType? projectUserRole) = await _projectService.GetProjectUserRole(entityId, _httpContextAccessor.HttpContext.GetUserId());
+        if (!projectStatus.Equals(ProjectStatusType.Active)
+          || !(projectUserRole.HasValue && projectUserRole.Value.Equals(ProjectUserRoleType.Manager))
+          && !await _accessValidator.HasRightsAsync(Rights.AddEditRemoveProjects))
+        {
+          return _responseCreator.CreateFailureResponse<List<Guid>>(HttpStatusCode.Forbidden);
+        }
+      }
+      else if (!await _accessValidator.HasRightsAsync(Rights.AddEditRemoveWiki)
```
Comment (vorobevamv) [inline src/FileService.Business/Commands/File/CreateFilesCommand.cs:69]: may be use switch/case here?

File: src/FileService.Business/Commands/File/CreateFilesCommand.cs [hunk 2]
```diff
@@ -61,7 +77,15 @@ public async Task<OperationResultResponse<List<Guid>>> ExecuteAsync(Guid entityI
 
       if (response.Body.Any())
       {
-        await _publish.CreateFilesAsync(entityId, access, response.Body);
+        if (serviceType == ServiceType.Project)
+        {
+          await _publish.CreateFilesAsync(entityId, access, response.Body);
+        }
+        else if (serviceType == ServiceType.Wiki)
```
Comment (vorobevamv) [inline src/FileService.Business/Commands/File/CreateFilesCommand.cs]: may be switch/case too?

File: src/FileService/appsettings.json [hunk 3]
```diff
@@ -30,6 +30,7 @@
   "RabbitMQ": {
     // FileService
```
Comment (vorobevamv) [inline src/FileService/appsettings.json]: rename to // Subscriptions

**Review by vorobevamv — APPROVED** [review]

**Review by ndoriomedov — APPROVED** [review]

File: src/FileService.Business/Commands/File/CreateFilesCommand.cs (hunk 2 above)
Comment (VLiamin) [reply src/FileService.Business/Commands/File/CreateFilesCommand.cs]: done

File: src/FileService/appsettings.json (hunk 3 above)
Comment (VLiamin) [reply src/FileService/appsettings.json]: done

**Pull request merged**